test/integration/dataset_collection.py

```diff
@@ -60,8 +97,28 @@ def test_dataset_collection(self):
                         f"Failed to run analyze function in file {full_filename}"
                     ) from e
 
-            new_results = []
+            new_results = pd.DataFrame(columns=[
+                'input_filename',
+                'class_name',
+                'invocation_text_string',
+                'method_where_invocation_occurred',
+                'start_line_of_function_where_invocation_occurred',
+                'invocation_method_name',
+                'invocation_method_start_line',
+                'invocation_method_end_line',
+                'can_be_parsed'
+            ])
             for x in results_output:
-                x[0] = Path(x[0]).name
-                new_results.append(x)
-        self.assertEqual(new_results.sort(), results_predefined.sort())
+                x['input_filename'] = str(Path(x['input_filename']).name)
+                del x['output_filename']
```

Comment:
is it make sense?

Reply:
of course. we do not need `output_filename` name cause it is generation changes often. And we do not need to check it